ldk/javascript/src/webpack/generate-banner.ts

```diff
@@ -1,16 +1,53 @@
-export function generateMetadata(): string {
+import { LoopPermissions } from './loop-permissions';
+
+function isPermissionDefined(permission: string[]): boolean {
```

Comment:
I think this function name is incorrect; should be `isPermissionUndefined` based on the logic I see below

Reply:
Good catch.